### ISSUE TITLE
Add support for e-Golf - tested and working with weconnect lib

### ISSUE THREAD

**Issue by ryanjdillon** [mitch-dc/volkswagen_we_connect_id#209]
**Is your feature request related to a problem? Please describe.**
I have a 2018 e-Golf in Norway with subscription to WeConnect ID services including climatization control, which I'd like to have supported with this integration.

I see that there are existing issues (#201 and #202), but it would make sense to me to have each model tracked separately in its own issue.

**Describe the solution you'd like**
Support e-Golf models

**Describe alternatives you've considered**
I've not considered alternatives

**Additional context**
I added `"e-Golf"` to `SUPPORTED_MODELS` in the integration `__init__.py`, and I was able to get things setup and sensor values read. 

It does not appear that the "control" services defined in the `__init__.py` and `services.yaml` are available. I tested the `weconnect` SDK, and I was able to activate/deactive climatization. 

I may just be missing some context here, but if you can point me in the right direction for things to check, I'd be happy to open a PR for adding "e-Golf" support.

**Comment by Maaxion** [comment]
This is now more relevant as We Connect ( the old app) is shutting down, which will mean that [homeassistant-volkswagencarnet](https://github.com/robinostlund/homeassistant-volkswagencarnet) will also stop working and e-golf (And others) owners will be left without a working HA integration

**Comment by guy-BW** [comment]
Yes please for this.. My daughter has just bought the e-golf and it would be great to add it to HA along with the ID.3 and ID.4.

I've made the manual adjustment as suggest above and it's working to retrieve the status.
